package.json

```diff
@@ -22,13 +22,14 @@
     "index.js"
   ],
   "scripts": {
-    "cover": "istanbul cover _mocha",
+    "coveralls": "nyc report --reporter=text-lcov | coveralls",
+    "test_coverage": "nyc --reporter=text mocha",
     "test": "mocha"
   },
   "devDependencies": {
     "coveralls": "^2.11.9",
-    "istanbul": "^0.4.3",
-    "mocha": "2.x"
+    "mocha": "2.x",
+    "nyc": "^12.0.2"
```

Comment:
`nyc` version 15.1.0 requires node.js >= `8.6` and on `4` and `6` it fails with

https://app.travis-ci.com/github/farhadi/node-smpp/jobs/559932321#L224-L228
https://app.travis-ci.com/github/farhadi/node-smpp/jobs/559932320#L224-L228

All versions after `13.0.1` require `find-up` version that does not support node.js `4`

https://app.travis-ci.com/github/farhadi/node-smpp/jobs/559933794#L301-L304

That is why currently I set the version to 12.0.2